models/event.py

```diff
@@ -226,7 +226,15 @@ def week(self):
             self._week = days / 7
 
         return self._week
-
+                          
+    @property
+    def week_str(self):
+        if self.week is None:
+            return None
+        if self.year == 2016:
+            return "Week {}".format(0.5 if self.week == 0 else self.week)
+        return "Week {}".format(self.week + 1)
+    
```

Comment:
`Blank line contains whitespace`
https://travis-ci.org/the-blue-alliance/the-blue-alliance/jobs/494044321#L3239